cmd/eudico/mirvalidator/run.go

```diff
@@ -81,7 +81,14 @@ var runCmd = &cli.Command{
 		&cli.IntFlag{
 			Name:  "segment-length",
 			Usage: "The length of an ISS segment. Must not be negative",
-			Value: 1,
+		},
+		&cli.DurationFlag{
+			Name:  "blocks-delay",
```

Comment:
I'd rather call this `max-block-delay`. It's more descriptive.

Reply:
Fixed